f5/bigip/sys/performance.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from f5.bigip.mixins import UnnamedResourceMixin
+from f5.bigip.resource import Collection
+from f5.bigip.resource import Resource
+from f5.bigip.resource import UnsupportedOperation
+
+
+class Performance(Collection):
+    def __init__(self, sys):
+        super(Performance, self).__init__(sys)
+        self._meta_data['allowed_lazy_attributes'] = [All_Stats]
+        self._meta_data['uri'] =\
+            self._meta_data['container']._meta_data['uri'] + "performance/"
```

Comment:
I think we should rename this class to "`Performances`" and remove the explicit handling of "`self._meta_data['uri']`" from its constructor since that should be handled automatically in the call to "`super`".